src/components/pages/products-list/ProductsGrid/ProductsGrid.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable max-len */
+import { productsData } from "../../../../mocks/products";
+import { categoriesData } from "../../../../mocks/product-categories";
+import { Styled } from "./Styled";
+
+import Card from "../../../common/Card";
+import Image from "../../../common/Image";
+import Title from "../../../common/Title";
+
+export function ProductsGrid({ activeFilters }) {
+  let activeList = [];
+  for (let filter in activeFilters) {
+    console.log(filter);
+    if (activeFilters[filter]) {
+      activeList.push(filter);
+    }
+  }
```

Comment:
The list can be obtained using `filter` and `map`, something like:

```suggestion
  const activeList = Object.entries(activeFilters)
    .filter(([filter, isChecked]) => isChecked)
    .map(([filter]) => filter);
```